### ISSUE TITLE
Can you provide me with a way to use it?

### ISSUE THREAD

**Issue by MinYoung-Lim** [jatinsha/sketch#1]
Hello. 
I tried to run this tool, but it didn't work properly.
Can you give me the tutorial and version for package?  ex) Python version, tkinter version, ...
I'd appreciate it if you could give me a requirement.txt.

Thank you.

**Comment by jatinsha** [comment]
1.	Please download/clone the repository from https://github.com/jatinsha/sketch.
2.	Go inside the ‘sketch’ directory and run the GUI tool using ‘python3 testSketch’
3.	A GUI will popup with a blank canvas [or with last working projects through history].
4.	In the Taskbar choose the desired Kernel (e.g.  ONNX). Make sure you have the specified toolchain (LuaTorch, PyTorch) installed in your system to use them.
5.	DRAW: Start drawing/sketching a model using “Drag and Drop menu” on the leftside. E.g. Drag Conv2d, BatchNorm2d and ReLU one by one. Perform a mouse down (click down on mouse) on the blue edges of these blocks which shall start a new connection (green color) and then drop that on another block to make a connection.
6.	DELETE: Select an element by left click, this will make them blue. You can also select multiple items creating a selection box using mouse. Use Ctrl+Delete to delete an element. 
7.	COMPILE: Once you connect Conv2d -> BatchNorm2d -> ReLU, you have a basic model ready. Hit “Compile” button and choose a file name (e.g. test.onnx). The tool will compile the sketch and save the onnx file. It will also show the output on a new text editor on right side.
8.	EDIT: Right click on any Layer/Block and its properties will show up in the “Node and Default Edit” window. You can edit them as desired.
9.	CUSTOM LAYERS: Drag the Yellow Sequential Block into the existing canvas. This will activate the “Group Canvas” window below. “Double right click” on Sequential Block that you just dragged on the canvas and Change the name to “ConvBNReLU”. Now whatever model you draw within the Group Canvas would go inside the custom ConvBNReLU Layer. You can edit the properties of individual layers like before and export them to say ConvBNReLU.sk to use them in future.
